templates/tableful.html

```diff
@@ -0,0 +1,30 @@
+{% extends "base.html" %}
+{% block content %}
+{% if tag %}
+<h3>Viewing tag: <a href="{{ url_for('index', tag=tag, page=1) }}">{{ tag }}</a></h3>
+{% endif %}
+<table>
+{% for quote in quotes %}
+    <tr>
+        <td style="padding-top: 2em;">{{ quote.text }}<small>{{ quote.author.name }}</small></td>
```

Comment:
Instead of using `<small>`, we could make it:

    {{ quote.text }}
    <br><br>
    Author: {{ quote.author.name }}